azure-pipelines-official.yml

```diff
@@ -171,3 +175,10 @@ jobs:
   - task: ms-vseng.MicroBuildTasks.521a94ea-9e68-468a-8167-6dcf361ea776.MicroBuildCleanup@1
     displayName: Perform Cleanup Tasks
     condition: succeededOrFailed()
+
+- template: /eng/common/templates/phases/publish-build-assets.yml
```

Comment:
Use the `/eng/common/templates/job/publish-build-assets.yml` template instead of the `phases/publish-build-assets.yml` template.  

I think you'll still hit an issue trying to queue your build, but we should be able to reauthorize your build definition and unblock you.